cognite/client/config.py

```diff
@@ -101,3 +102,29 @@ def __str__(self) -> str:
 
     def _repr_html_(self) -> str:
         return str(self)
+
+    @classmethod
+    def create_default(
```

Comment:
Just `default`, to stay consitent with the method on CogniteClient?
```suggestion
    def default(
```